plugins/playstation/src/playstation-header.vala

```diff
@@ -0,0 +1,95 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.PlayStationHeader : Object {
+	private const size_t[] OFFSETS = { 0xD368, 0xD3A8 };
+	private const string[] IDS = { "SLUS", "SCUS", "SLES", "SCES", "SLPS", "SLPM", "SCPS" };
+	private const string MAGIC_VALUE = "BOOT";
+
+	private File file;
```

Comment:
Fields go after properties (I'm not sure it's specified in CodingStyle.txt... it sould though).